api/types/version.go

```diff
@@ -0,0 +1,4 @@
+package types
+
+// APIVersion is the current API version.
+const APIVersion = "1.0"
```

Comment:
Should this be set to the latest MicroCloud release `1.1`?

Reply:
I don't believe we are making any changes to the MicroCloud API in this release so no.

Reply:
This is the API version that is in the path prefix: like `127.0.0.1:9000/1.0`

Reply:
True, I somehow mixed it with the versioning @MggMuggins is currently working on in https://github.com/canonical/microcluster/pull/165.